code/edgebridge/src/androidTest/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeFunctionalTests.java

```diff
@@ -96,19 +97,21 @@ public void testTrackStateSendsEdgeExperienceEvent() throws InterruptedException
 		assertEquals("analytics.track", requestData.get("events[0].xdm.eventType"));
 		assertNotNull(requestData.get("events[0].xdm.timestamp"));
 		assertNotNull(requestData.get("events[0].xdm._id"));
-		assertEquals("Test State", requestData.get("events[0].data.state"));
-		assertEquals("testValue", requestData.get("events[0].data.contextdata.testKey"));
+		assertEquals("state name", requestData.get("events[0].data.__adobe.analytics.pageName"));
+		assertEquals("propValue1", requestData.get("events[0].data.__adobe.analytics.c1"));
+		assertEquals("value1", requestData.get("events[0].data.__adobe.analytics.contextData.key1"));
```

Comment:
I wish we could have used the cute json payload comparison here 💭 

For these tests can you also assert on the exact count expected.

Reply:
I'd say don't assert on those, just the total count and strict asserts on the data fields you already covered